target/linux/ramips/dts/rt5350_dlink_dir-506l.dts

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,dir-506l", "ralink,rt5350-soc";
+	model = "D-Link DIR-506L";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_red;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		usb {
+			label = "dir-506l:green:usb";
```

Comment:
Model (dir-506l:) can be dropped from LED label now, so you can just keep "green:usb" etc.